packages/blade/src/components/Checkbox/__tests__/__snapshots__/CheckboxGroup.web.test.tsx.snap

```diff
@@ -209,7 +209,7 @@ exports[`<CheckboxGroup /> should render with help text 1`] = `
   >
     <span
       id="checkbox-group-49-label-54"
-      style="width: auto; flex-shrink: 0; margin-right: 16px; word-break: break-all;"
+      style="width: auto; flex-shrink: 0; word-break: break-all;"
```

Comment:
When the label is on top on web, we used to add a margin-bottom as well as margin-right. We do not need a right margin when the label is on top. Removed the unnecessary margin which led to a snapshot change.